include/xlsxwriter/chart.h

```diff
@@ -523,6 +523,20 @@ typedef enum lxw_chart_axis_label_position {
     LXW_CHART_AXIS_LABEL_POSITION_NONE
 } lxw_chart_axis_label_position;
 
+/**
+ * @brief Axis label alignments.
+ */
+typedef enum lxw_chart_axis_label_aligment {
+    /** Series data label alignment: center. */
+    LXW_CHART_LABEL_ALIGNMENT_CENTER,
```

Comment:
Maybe I should those `LXW_CHART_AXIS_LABEL_ALIGNMENT_...` to be consistent with the function name and other properties for axis.

Reply:
I wouldn't worry about it. I'll write the C code once I do the Python and Perl versions.

PRs to this codebase don't help me a lot. Although this is better than usual.

Thanks.